.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
```

Comment:
Should we have an "external reference" field, that can hold either a JIRA number or a ZenDesk tickert ref?

Reply:
I dont think that I can add it to the auto-generated list of labels, but we can definitely add it to the end.

something like:?

External Reference
ZenDesk/JIRA numbers if available

Reply:
Perfect! :)